<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorChooser</class>
 <widget class="QMainWindow" name="ColorChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>489</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ColorChooser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="l_filter">
          <property name="text">
           <string>Filter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_filter"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectColumns</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_showEditor">
        <property name="text">
         <string>Show Editor</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ColorEditor" name="w_colorEditor" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>343</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCopy">
    <property name="title">
     <string>Copy</string>
    </property>
    <addaction name="actionCopy_As_Hex_RGB"/>
    <addaction name="actionCopy_As_Hex_ARGB"/>
    <addaction name="actionCopy_As_Byte_RGB"/>
    <addaction name="actionCopy_As_Float_RGB"/>
    <addaction name="actionCopy_As_UE4_Float"/>
   </widget>
   <widget class="QMenu" name="menuPaste">
    <property name="title">
     <string>Paste</string>
    </property>
    <addaction name="actionPaste_From_Float_RGB"/>
    <addaction name="actionPaste_From_Byte_RGB"/>
    <addaction name="actionPaste_From_UE4_Float"/>
   </widget>
   <widget class="QMenu" name="menuConvert">
    <property name="title">
     <string>Convert</string>
    </property>
    <addaction name="actionByteToFloat"/>
    <addaction name="actionFloatToByte"/>
    <addaction name="separator"/>
    <addaction name="actionHexToFloat"/>
    <addaction name="actionFloatToHex"/>
    <addaction name="separator"/>
    <addaction name="actionHexToByte"/>
    <addaction name="actionByteToHex"/>
   </widget>
   <addaction name="menuCopy"/>
   <addaction name="menuPaste"/>
   <addaction name="menuConvert"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCopy_As_Hex_RGB">
   <property name="text">
    <string>as Hex RGB</string>
   </property>
   <property name="toolTip">
    <string>as Hex RGB</string>
   </property>
  </action>
  <action name="actionCopy_As_Hex_ARGB">
   <property name="text">
    <string>as Hex ARGB</string>
   </property>
  </action>
  <action name="actionCopy_As_Byte_RGB">
   <property name="text">
    <string>as Byte RGB</string>
   </property>
  </action>
  <action name="actionCopy_As_Float_RGB">
   <property name="text">
    <string>as Float RGB</string>
   </property>
  </action>
  <action name="actionPaste_From_Float_RGB">
   <property name="text">
    <string>from Float RGB</string>
   </property>
  </action>
  <action name="actionByteToFloat">
   <property name="text">
    <string>Byte → Float</string>
   </property>
   <property name="toolTip">
    <string>Byte → Float</string>
   </property>
  </action>
  <action name="actionFloatToByte">
   <property name="text">
    <string>Float → Byte</string>
   </property>
   <property name="toolTip">
    <string>Float → Byte</string>
   </property>
  </action>
  <action name="actionHexToFloat">
   <property name="text">
    <string>Hex → Float</string>
   </property>
   <property name="toolTip">
    <string>Hex → Float</string>
   </property>
  </action>
  <action name="actionFloatToHex">
   <property name="text">
    <string>Float → Hex</string>
   </property>
   <property name="toolTip">
    <string>Float → Hex</string>
   </property>
  </action>
  <action name="actionHexToByte">
   <property name="text">
    <string>Hex → Byte</string>
   </property>
   <property name="toolTip">
    <string>Hex → Byte</string>
   </property>
  </action>
  <action name="actionByteToHex">
   <property name="text">
    <string>Byte → Hex</string>
   </property>
   <property name="toolTip">
    <string>Byte → Hex</string>
   </property>
  </action>
  <action name="actionPaste_From_Byte_RGB">
   <property name="text">
    <string>from Byte RGB</string>
   </property>
  </action>
  <action name="actionCopy_As_UE4_Float">
   <property name="text">
    <string>as UE4 Float</string>
   </property>
  </action>
  <action name="actionPaste_From_UE4_Float">
   <property name="text">
    <string>from UE4 Float</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEditor</class>
   <extends>QWidget</extends>
   <header>ColorEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
